go/vt/wrangler/materializer.go

```diff
@@ -583,46 +588,77 @@ func (wr *Wrangler) buildMaterializer(ctx context.Context, ms *vtctldatapb.Mater
 }
 
 func (mz *materializer) deploySchema(ctx context.Context) error {
+
 	return mz.forAllTargets(func(target *topo.ShardInfo) error {
-		for _, ts := range mz.ms.TableSettings {
-			tableSchema, err := mz.wr.GetSchema(ctx, target.MasterAlias, []string{ts.TargetTable}, nil, false)
+		allTables := []string{"/.*/"}
+
+		hasTargetTable := map[string]bool{}
+		{
```

Comment:
Anonymous blocks are not something we've used in vitess before. I think it's ok because it doesn't hamper readability. I'd feel better if you can show me similar idiomatic usage in other authoritative source, like the go libraries.

Reply:
i don't mind pulling it out, but just did it to keep variables clean, to show what we actually care about below. but if you insist, i can pull them out. i just worried about this fn being relatively long, and not knowing which vars are pertinent.